documentation/src/main/asciidoc/topics/con_file_based_cache_stores.adoc

```diff
@@ -0,0 +1,77 @@
+[id='file-stores_{context}']
+= File-Based Cache Stores
+
+File-based cache stores provide persistent storage on the local host filesystem where {brandname} is running.
+For clustered caches, file-based cache stores are unique to each {brandname} node.
+
+You should use file-based cache stores to persist data that overflows from memory when the data container exceeds size or time restrictions.
```

Comment:
Not sure what you mean by time restrictions. Expiration will remove from stores as well. Stores can also be used even when data doesn't overflow ( overflow to me implies passivation).

Reply:
Yeah, let's just remove this sentence.